src/test/kotlin/net/atos/zac/signalering/model/SignaleringFixtures.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * SPDX-FileCopyrightText: 2024 Lifely
+ * SPDX-License-Identifier: EUPL-1.2+
+ */
+
+package net.atos.zac.signalering.model
+
+import net.atos.client.zgw.zrc.model.Zaak
+import net.atos.client.zgw.zrc.model.createZaak
+
+fun createSignalering(
+    zaak: Zaak = createZaak()
+) = Signalering().apply {
+    this.type = SignaleringType().apply {
```

Comment:
There's no need to use `this` in the context of the `apply` code block

Reply:
Fixed